shared/desktop/app/main-window.js

```diff
@@ -62,7 +63,10 @@ export default function() {
 
   const openedAtLogin = app.getLoginItemSettings().wasOpenedAtLogin
   const isRestore = getenv.boolish('KEYBASE_RESTORE_UI', false) || app.getLoginItemSettings().restoreState
-  const hideWindowOnStart = getenv.string('KEYBASE_START_UI', '') === 'hideWindow'
+  // Start not shown by default on Windows
+  const hideWindowOnStart =
```

Comment:
lets discuss on slack. i believe this variable should be exactly the same logic on all platforms so i don't see why this isn't correct on windows.